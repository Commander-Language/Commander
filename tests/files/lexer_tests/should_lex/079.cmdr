//Tests commands inside of strings
print $"Output: {`cat myfile.txt`[0]}!";