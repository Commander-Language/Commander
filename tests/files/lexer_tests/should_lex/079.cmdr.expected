PRINT 'print' 2:1
STRINGVAL '"' 2:7
STRINGLITERAL 'Output: ' 2:9
BACKTICK '`' 2:18
CMDSTRINGVAL 'cat' 2:19
CMDSTRINGVAL 'myfile.txt' 2:23
BACKTICK '`' 2:33
LSQUARE '[' 2:34
INTVAL '0' 2:35
RSQUARE ']' 2:36
STRINGLITERAL '!' 2:38
ENDSTRINGVAL '"' 2:39
SEMICOLON ';' 2:40
END_OF_FILE 2:41
